build.gradle.kts

```diff
@@ -10,7 +10,7 @@ plugins {
 
 allprojects {
   group = "uk.gov.justice.service.hmpps"
-  version = "1.1.7"
+  version = "1.1.8-wip"
```

Comment:
Not planning to publish until all work is complete - but I will endeavour to keep the library in a state where it can be published (in case we need an emergency fix or security issue).